src/Core/Extensions/Basic.cs

```diff
@@ -272,12 +284,12 @@ public static void RunProxy(this IApplicationBuilder app, Action<IProxyBuilder>
         /// <param name="httpProxyOptions">The HTTP options.</param>
         /// <param name="wsProxyOptions">The WS options.</param>
         /// <returns>A <see cref="Task"/> which completes when the request has been successfully proxied and written to the response.</returns>
-        public static Task ProxyAsync(this ControllerBase controller, string httpEndpoint, string wsEndpoint, HttpProxyOptions httpProxyOptions = null, WsProxyOptions wsProxyOptions = null)
+        public static async Task ProxyAsync(this ControllerBase controller, string httpEndpoint, string wsEndpoint, HttpProxyOptions httpProxyOptions = null, WsProxyOptions wsProxyOptions = null)
         {
             var httpProxy = new HttpProxy((c, a) => new ValueTask<string>(httpEndpoint), httpProxyOptions);
             var wsProxy = new WsProxy((c, a) => new ValueTask<string>(wsEndpoint), wsProxyOptions);
             var proxy = new Builders.Proxy(null, httpProxy, wsProxy);
-            return controller.HttpContext.ExecuteProxyOperationAsync(proxy);
```

Comment:
Might want to show the throwaway result.

Reply:
Yeah I wasn't sure on this one, didn't want to change the API but the user of this API can still choose to ignore the result and it should be their choice, an Enum would be the best output for them